sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        name=f"retail-computer-vision-{datetime.now().strftime('%Y%m-%d%H-%M%S')}",
</pre><p> </p><p>FYI, there's a helper function in sagemaker to generate the unique name with timestamp and UUID. But the current solution works well also</p><pre class="ql-syntax" spellcheck="false">from sagemaker.utils import unique_name_from_base
name = unique_name_from_base("retail-computer-vision")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Great tip! I was not aware that we have this functionality. I'll update it to use the sagemaker function because I think that makes more sense.